.github/actions/setup/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Set up project'
```

Comment:
This re-usable action should allow us to DRY up the setup slightly for the other builds